translations/README.spanish.md

```diff
@@ -0,0 +1,216 @@
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://eepurl.com/dBUfJ5"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura GraphQL Engine es un servidor GraphQL ultrarrápido que le ofrece **API GraphQL instantáneas y en tiempo real sobre Postgres**, con [**activadores de webhook**](../event-triggers.md) en eventos de base de datos y [**remoto esquemas**](../remote-schemas.md) para lógica de negocios.
+
+Hasura lo ayuda a construir aplicaciones GraphQL en PostgreSQL o migrar gradualmente a GraphQL para aplicaciones existentes que usan Postgres.
+
+Lea más en [hasura.io](https://hasura.io) y en [docs](https://docs.hasura.io).
+
+------------------
+
+![Demostración del motor Hasura GraphQL](../assets/demo.gif)
+
+------------------
+
+![Hasura GraphQL Engine Realtime Demo](../assets/realtime.gif)
+
+-------------------
+
+## Caracteristicas
+
+* **Realice consultas potentes**: filtrado incorporado, paginación, búsqueda de patrones, inserción masiva, actualización, eliminación de mutaciones
+* **Realtime**: Convierta cualquier consulta GraphQL en una consulta en vivo mediante suscripciones
+* **Combinar esquemas remotos**: Acceda a esquemas personalizados de GraphQL para la lógica empresarial a través de un único punto final de GraphQL Engine. [**Leer más**](../remote-schemas.md).
+* **Activar webhooks o funciones sin servidor**: En Postgres, insertar/actualizar/eliminar eventos ([leer más](../event-triggers.md))
+* **Funciona con bases de datos en vivo existentes**: apúntelo a una base de datos de Postgres existente para obtener instantáneamente una API GraphQL lista para usar
+* **Control de acceso de grano fino**: control de acceso dinámico que se integra con su sistema de autenticación (por ejemplo: auth0, firebase-auth)
+* **Alto rendimiento y baja huella**: ~ 15 MB de imagen acoplable; ~50MB RAM @ 1000 req/s; multinúcleo consciente
+* **UI de administrador y migraciones**: UI de administrador y migraciones de esquemas inspiradas en Rails
+* **Postgres** ❤️: admite los tipos de Postgres (PostGIS/geo-location, etc.), convierte las vistas en *gráficos*, activa funciones almacenadas o procedimientos con mutaciones
+
+Lea más en [hasura.io](https://hasura.io) y en [docs](https://docs.hasura.io).
+
+## Tabla de contenido
+<!-- markdown-toc start - No edite esta sección. Ejecute M-x markdown-toc-refresh-toc -->
+**Tabla de contenido**
+
+- [Inicio rápido:](#inicio-rápido)
+    - [Implementación con un clic en Heroku](#despliegue-con-un-clic-en-heroku)
+    - [Otros métodos de implementación](#otros-métodos-de-implementación)
+- [Arquitectura](#arquitectura)
+- [Herramientas del lado del cliente](#herramientas-del-lado-del-cliente)
+- [Agregar lógica de negocios](#añadir-lógica-de-negocios)
```

Comment:
Here it's `Agregar` and below in the title it's `Añadir`. Could you fix this please?

Reply:
Done mam